src/lib/GClient.ts

```diff
@@ -22,17 +30,88 @@ export enum AutoDeferType {
 	'UPDATE' = 3,
 }
 
+/**
+ * Options for the GClient.
+ * @extends {ClientOptions}
+ */
 export interface GClientOptions extends ClientOptions {
+	/**
+	 * Support for message commands.
+	 * @type {boolean}
+	 */
 	messageSupport?: boolean;
+	/**
+	 * Prefix for message commands
+	 * @requires {@link GClientOptions.messageSupport} to be enabled
+	 * @type {string}
+	 */
 	messagePrefix?: string;
+	/**
+	 * Whether to send a message for an unknown command.
```

Comment:
This should state that it only works for message commands. 
```suggestion
	 * Whether to send a message for an unknown message command.
```